index.html

```diff
@@ -0,0 +1,214 @@
+                <li class="right-nav-link">
+                    <a href="#main-program">Program</a>
+                </li>
+                <li class="right-nav-link">
+                    <a href="#feature-speaker">Speaker</a>
+                </li>
+                <li class="right-nav-link">
+                    <a href="#partner">Partner</a>
+                </li>
+                <li class="right-nav-link most-right-link">
+                    <a href="#">Tech Campion</a>
+                </li>
+            </ul>
+        </div>
+    </header>
+
+    <section class="mobile-nav">
+        <h1></h1>
+        <div class="close-mob-menu mob-side-nav">
+            <img src="./assets/icons/close-icon.png" alt="">
+        </div>
+        <ul class="mob-menu-list">
+            <li class="mob-side-nav">
+                <a href="#"  class="mob-menu-item" id="home">
+                    Home 
+                    <img src="./assets/icons/chevron_right.png" alt="">
+                </a>
+            </li>
+            <li class="mob-side-nav">
+                <a href="./page/about.html"  class="mob-menu-item" id="about">
+                    About 
+                    <img src="./assets/icons/chevron_right.png" alt="">
+                </a>
+            </li>
+            <li class="mob-side-nav">
+                <a href="#main-program"  class="mob-menu-item" id="sponsor">
+                    Main Programe
+                    <img src="./assets/icons/chevron_right.png" alt="">
+                </a>
+            </li>
+            <li class="mob-side-nav">
+                <a href="#feature-speaker"  class="mob-menu-item" id="join">
+                    Featured Speaker 
+                    <img src="./assets/icons/chevron_right.png" alt="">
+                </a>
+            </li>
+            <li class="mob-side-nav">
+                <a href="#partner"  class="mob-menu-item" id="news">
+                    Partners 
+                    <img src="./assets/icons/chevron_right.png" alt="">
+                </a>
+            </li>
+        </ul>
+    </section>
+
+    <section class="intro-container">
+        <header>
+            <span class="mobile-menu mob-side-nav">
+                <img src="./assets/icons/menuIcon.svg" alt="Humburgur Menu">
+            </span>
+    
+        </header>
+
+        <section class="home-intro">
+
+            <div class="page-intro">
+                <h3>"Hello Tech World"</h3>
+                <h1 id="title-desc">International Technology Talk SUMMIT 2022</h1>
+                <div class="opening-desc">
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quibusdam porro ab est reprehenderit reiciendis corporis excepturi nam perspiciatis non recusandae dolorem ut, nisi aperiam quis quidem veritatis qui a nostrum?</p>
+                </div>
+                <h1 id="event-date">2023-02-15(Thur) ~ 16(Fri)</h1>
+                <h4>@ Ambassdar Hotel, Urban Center, Addis Ababa</h4>
```

Comment:
- [ ] For semantic HTML, there shouldn't be more than one `h1` tag per page. The `h1` tag should be used for the _**main header**_ of a page. Kindly replace all other `h1` tags on this page and the _**about**_ page with other `hx` tags.

Reply:
Noted and solved